internal/backend/backend.go

```diff
@@ -178,6 +179,13 @@ func newMetrics(registry prometheus.Registerer) *metrics {
 			},
 			[]string{"code"},
 		),
+		retriesTotal: prometheus.NewGauge(
+			prometheus.GaugeOpts{
+				Namespace: "kubernetes_scanner",
+				Name:      "backend_unreconciled_resources_total",
```

Comment:
Please shout if you have a better idea for a name. This is a bit long imo... 

The same goes for the Help text. How do we best describe this (concisely)?

Reply:
```suggestion
				Name:      "unreconciled_resources_total",
```

I don't think the length is so much of an issue, but this metric isn't "backend"-specific - it's the number of as-yet unreconciled resources. WDYT?

For the help, how about "the number of unreconciled resources that are still in backoff-retry loops"?